creation/cloud-init.md

```diff
@@ -615,3 +626,120 @@ The metadata will be available in the guests config drive `openstack/latest/meta
 }
 ```
 
+Sysprep Examples
+-------------------
```

Comment:
I believe that the new markdown renderer for the user guide expects headers in this form instead:
```suggestion
## Sysprep Examples
```

@mazzystr can you confirm?

Reply:
Let's stick with the docsify header style until we are fully migrated to mkdocs